Makefile

```diff
@@ -33,5 +37,20 @@ endif
 	setfacl -m other::rwx ./tmp/nexus-volume
 	$(CACHITO_COMPOSE_ENGINE) up
 
-test:
-	PATH="$PWD/venv/bin:$PATH" tox
+run-down run-stop:
+	$(CACHITO_COMPOSE_ENGINE) down $(DOWN_OPTS)
+
+run-build: run-down
+	$(CACHITO_COMPOSE_ENGINE) build
+
+# Keep test target for backwards compatibility
+test test-unit:
+	PATH="${PWD}/venv/bin:${PATH}" tox
+
+test-integration:
+	PATH="${PWD}/venv/bin:${PATH}" tox -e integration
+
+test-suite:
+	PATH="${PWD}/venv/bin:${PATH}" tox -e $(PYTHON_VERSION_TOX) -- $(PYTHON_TEST_SUITE)
```

Comment:
Maybe `PYTHON_TEST_SUITE` should be called something like `TOX_ARGS` because you could use this to run something like:
```
tox -e py38 -- -x --no-cov tests/test_workers/test_tasks/test_gomod.py
```

Same for `PYTHON_VERSION_TOX`. That should probably be something like `TOX_ENVLIST` ?

Reply:
Addressed.